src/git.ts

```diff
@@ -827,11 +873,22 @@ export function handleDiffFiles(actionData: ActionData): string | undefined {
     return;
 }
 
-export function handleGitLog(_actionData: ActionData): string | undefined {
+export function handleGitLog(actionData: ActionData): string | undefined {
     assert(repo);
 
+    const logLimit: number | undefined = actionData.params?.log_limit;
+
     repo.log().then((commits: Commit[]) => {
-        NEURO.client?.sendContext(`Commit log: ${JSON.stringify(commits)}`);
+        // If log_limit is defined, restrict number of commits to that value.
+        if (logLimit) {
+            commits = commits.slice(0, logLimit - 1);
```

Comment:
The `end` parameter of `slice` is exclusive.
```suggestion
            commits = commits.slice(0, logLimit);
```